II.pass-tests/11-cities-formatter/cities.js

```diff
@@ -0,0 +1,18 @@
+
+var capitals = [
+    { city: 'Paris', country: 'France' },
+    { city: 'Madrid', country: 'Spain' },
+    { city: 'Rome', country: 'Italy' },
+];
+function  formatCities(capitals,func){
+return capitals.map(capital=>{
```

Comment:
Once you've looked into "destructuring" in the ES6 part, try rewriting this using that feature.